homeworks/04-css/index.html

```diff
@@ -0,0 +1,138 @@
+                            alt="logo of twitter"></a>
+                    <a href="#"><img class="header-intro-icon" src="./assets/logo-linkedin.svg"
+                            alt="logo of linkedin"></a>
+                    <a href="#"><img class="header-intro-icon icon-yt" src="./assets/logo-yt.svg"
+                            alt="logo of youtube"></a>
+                </div>
+            </div>
+        </header>
+        <ul class="content-list">
+            <li class="content-list-item">
+                <h2 class="content-list-item-title">7</h2>
+                <p>Years of work experience</p>
+            </li>
+            <li class="content-list-item">
+                <h2 class="content-list-item-title">50+</h2>
+                <p>Completed projects</p>
+            </li>
+            <li class="content-list-item">
+                <h2 class="content-list-item-title">20+</h2>
+                <p>Satisfied customers</p>
+            </li>
+        </ul>
+        <div class="content-btns-cv-contact">
+            <button class="btn-cv btns-cv-contact">Download CV
+                <img src="./assets/VectorCV.svg" alt="arrow download">
+            </button>
+            <button class="btn-contact btns-cv-contact">Contact me</button>
+        </div>
+        <div class="content-btns-portfolio-skills">
+            <button class="btn-portfolio btns-portfolio-skills active">Portfolio</button>
+            <button class="btn-skills btns-portfolio-skills">Skills</button>
+        </div>
+        <main>
+            <ul class="content-main-slides">
+                <li>
+                    <img class="content-main-slide-pic" src="./assets/icon1.svg" alt="first slide">
+                    <div class="content-main-slide">
+                        <h3 class="content-main-slide-title">SmarterTrading411</h3>
+                        <span>Landing page of SmarterTrading411 created using wordPress</span>
+                        <a class="main-slides-icon_share" href="#"><img src="./assets/share.svg"
+                                alt="icon to share"></a>
+                    </div>
+                </li>
+                <li>
+                    <img class="content-main-slide-pic" src="./assets/icon2.svg" alt="second slide">
+                    <div class="content-main-slide">
+                        <h3 class="content-main-slide-title">SmarterTrading411</h3>
+                        <span>Landing page of SmarterTrading411 created using wordPress</span><a
+                            class="main-slides-icon_share" href="#"><img src="./assets/share.svg"
+                                alt="icon to share"></a>
+                    </div>
+                </li>
+                <li><img class="content-main-slide-pic" src="./assets/icon3.svg" alt="third slide">
+                    <div class="content-main-slide">
+                        <h3 class="content-main-slide -title">SmarterTrading411</h3>
```

Comment:
Classes naming not correctly follows BEM approach. Please investigate BEM closer.